layers/+lang/java/funcs.el

```diff
@@ -68,7 +68,8 @@
 (defun spacemacs//java-setup-ensime ()
   "Setup ENSIME."
   ;; jump handler
-  (add-to-list 'spacemacs-jump-handlers 'ensime-edit-definition)
+  (add-to-list 'spacemacs-jump-handlers-java-mode 'ensime-edit-definition)
+  (add-to-list 'spacemacs-jump-handlers-scala-mode 'ensime-edit-definition)
```

Comment:
You probably need to put this in the scala layer somewhere since the list will be undefined if that layer isn't declared.

Reply:
@jcazevedo https://github.com/jcazevedo/spacemacs/pull/1

Reply:
Thanks @szdavid92 for your commit and sorry for leaving this hanging.